fractal_server/app/routes/admin/v2.py

```diff
@@ -307,3 +314,126 @@ async def flag_task_v1_as_v2_compatible(
     await db.close()
 
     return Response(status_code=status.HTTP_200_OK)
+
+
+class TaskV2Minimal(BaseModel):
+
+    id: int
+    name: str
+
+    type: str
+    command_non_parallel: Optional[str] = None
+    command_parallel: Optional[str] = None
+
+    source: str
+
+    meta_non_parallel: dict[str, Any] = Field(default_factory=dict)
+    meta_parallel: dict[str, Any] = Field(default_factory=dict)
+
+    owner: Optional[str]
+    version: Optional[str]
+
+    docs_info: Optional[str] = None
+    docs_link: Optional[HttpUrl] = None
+
+    input_types: dict[str, bool] = Field(default_factory=dict)
+    output_types: dict[str, bool] = Field(default_factory=dict)
+
+
+class ProjectUser(BaseModel):
+    id: int
+    email: EmailStr
+
+
+class TaskContext(BaseModel):
+
+    task_v2_minimal: TaskV2Minimal
+
+    workflow_id: Optional[int]
```

Comment:
I would say that none of this attributes is `Optional`.
The only one that may be missing (even if it's by mistake, ref #1463) is `project_users`, but then we should simply have an empty list when we populate this model.
The other attributes (workflow and project id/name) cannot be missing.